Makefile

```diff
@@ -31,6 +31,10 @@ clean:
 distclean: clean
 	find vendor/ -maxdepth 1 -mindepth 1 -not -name vendor.json -exec rm {} -rf \;
 	rm -f manifest/*.yaml
+	rm -f .glide.status
+
+checksync: .glide.status
+	test -d vendor || make sync
 
 sync:
 	glide install --strip-vendor
```

Comment:
This is correct. `glide install` is cheaper and uses a cache. `glide update` crawls all deps and refreshes `glide.lock`.